src/game.cpp

```diff
@@ -11217,29 +11218,30 @@ void game::start_hauling( const tripoint &pos )
 }
 
 cata::optional<tripoint> game::find_or_make_stairs( map &mp, const int z_after, bool &rope_ladder,
-        bool peeking )
+        bool peeking, const tripoint pos )
```

Comment:
```suggestion
        bool peeking, const tripoint &pos )
```
Last changes, I promise :P
Clang-tidy is pretty strict about const qualified parameters now, passing by reference will get around that